pages/linux/gsettings.md

```diff
@@ -0,0 +1,32 @@
+# gsettings
+
+> Query and modify dconf settings with schema validation.
+> More information: <https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/using_the_desktop_environment_in_rhel_8/configuring-gnome-at-low-level_using-the-desktop-environment-in-rhel-8#using-gsettings-command_configuring-gnome-at-low-level>.
+
+- Set the value of a key. Fails if the key doesn't exist or the value is out of range:
+
+`gsettings set {{org.example.schema}} {{example-key}} {{value}}`
+
+- Print the value of a key or the schema-provided default if the key has not been set in `dconf`:
+
+`gsetings get {{org.example.schema}} {{example-key}}`
+
+- Un-set a key, so that its schema default value will be used:
```

Comment:
Sorry to be a pain!

```suggestion
- Unset a key, so that its schema default value will be used:
```

* https://www.collinsdictionary.com/dictionary/english/unset
* https://www.lexico.com/definition/unset
* https://www.merriam-webster.com/dictionary/unset

"Unset" is one word! Other than that I think this PR looks great!